deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: UnifiedPushServer
+metadata:
+  name: example-ups-with-backups
+spec:
+  backups:
+    # Note that backups here is a list, you can specify more than one
+    -
+      # REQUIRED: The name of a backup here will be the name of the
+      # generated CronJob
+      name: ups-daily-at-midnight
```

Comment:
why the name is required here? Why it is not generated programmatically? (e.g you can use the intance.Name)

Reply:
This supports a list of backups, rather than just one, which means that if we use `instance.Name`, we'd also need to have a generated part (or accept a suffix here). I think it's nicer to allow humans to give a name that can have some relation to what kind of backup it is (either based on what it's backing up (database, resources, etc) or the schedule, or whatever). Also, this is mostly copying the keycloak-operator.

Reply:
I am closing it. yes is required the namespace you are correct. we need to do it in the MSS Oper too